content/applications/inventory_and_mrp/purchase/manage_deals/manage.rst

```diff
@@ -4,226 +4,249 @@ Manage vendor bills
-   any stockable products.
+.. image:: manage/manage-configuration-settings.png
+   :align: center
+   :alt: Bill control policies in purchase app settings.
 
-4. Upon receiving a **Vendor Bill** from your Vendor, validate the bill
-   with products received in the previous step to ensure accuracy.
+The policy selected will be the default for any new product created. The definition of each policy
+is as follows:
 
-This process may be done by three different people within the company,
-or only one.
+- :guilabel:`Ordered quantities`: creates a vendor bill as soon as a purchase order is confirmed.
+  The products and quantities in the purchase order are used to generate a draft bill.
+- :guilabel:`Received quantities`: a bill is only created **after** part of the total order has been
+  received. The products and quantities **received** are used to generate a draft bill.
 
-Configuration
-=============
+.. tip::
+   If a product needs a different control policy, the default bill control policy can be overridden
+   by going to the :guilabel:`Purchase` tab in a product's template, and modifying its
+   :guilabel:`Control Policy` field.
 
-Installing the Purchase and Inventory applications
---------------------------------------------------
+.. image:: manage/manage-product-form.png
+   :align: center
+   :alt: Control policy field on product form.
 
-From the **Apps** application, search for the **Purchase** module and
-install it. Due to certain dependencies, installing Purchase will
-automatically install the **Inventory** and **Accounting** applications.
+3-way matching
+--------------
 
-Creating products
------------------
+*3-way matching* ensures vendor bills are only paid once some (or all) of the products included in
+the purchase order have actually been received.
 
-Creating products in Odoo is essential for quick and efficient
-purchasing within Odoo. Simply navigate to the Products submenu under
-Purchase and click create.
+To activate it, go to :menuselection:`Purchase app --> Configuration --> Settings`, and scroll down
+to the :guilabel:`Invoicing` section. Then, check the box next to :guilabel:`3-way matching:
+purchases, receptions, and bills`, and click :guilabel:`Save` to save changes.
 
-.. image:: manage/manage01.png
-  :align: center
+.. important::
+   :guilabel:`3-way matching` is **only** intended to work with the :guilabel:`Bill Control` policy
+   set to :guilabel:`Received quantities`.
 
-When creating the product, pay attention to the **Product Type** field,
-since it is important:
+   .. image:: manage/manage-three-way-matching.png
+      :align: center
+      :alt: Activated three-way matching feature in purchase settings.
 
-- Products that are set as **Stockable or Consumable** will allow you
-  to keep track of their inventory levels. These options imply
-  stock management and will allow for receiving these kinds of
-  products.
+Create (and manage) vendor bills on receipts
```

Comment:
Parenthesis are not needed here.
They're also not present on the other "create and manage" heading that appears later in the doc.
```suggestion
Create and manage vendor bills on receipts
```